lib/generators/active_record/install_generator.rb

```diff
@@ -28,6 +28,13 @@ def copy_migrations_and_model
         migration_template 'create_scores_and_points.rb',
                            'db/migrate/create_scores_and_points.rb'
       end
+
+      def migration_version
+       if Rails.version.start_with? '5'
+         "[#{Rails::VERSION::MAJOR}.#{Rails::VERSION::MINOR}]"
+       end
```

Comment:
Thank you, Nimir! We can assume we are always on `5`, as `master` only targets that version. Until we have Rails 6, we shouldn't need this conditional.

Reply:
alright, thought we needed backward compatibility with older rails versions! but yeah, if master is only for Rails > 5 then we can skip the check!